### PR TITLE
Fixes #3376, letters in review going out of view

### LINKED ISSUE #3376: Letters in Peer Review
**Summary:** 
Some words doesn't appear correctly to the user . The line <VIOLATION> under the <NO, COPYRIGHT> and some of the descriptions.

**Steps to reproduce:** 
We could change the dimensions.

**Device and Android version:** 
Redmi note 5 , Android version 9

**Commons app version:** 
2.12.3

**Screen-shots:** 
Screenshot_2020-02-05-19-40-04-954_fr free nrw commons

**Would you like to work on the issue?**
No

### PR DESCRIPTION
**Pull request by GearGit**
**Description**

Fixes #3376 Letters in Peer Review, the ReviewImageFragment was changed where the string for yes and no button text was changed to simple yes or no.

**Tests performed (required)**

Tested on ProdDebug.

**Screenshots (for UI changes only)**
Screenshot_20200320-001525_Commons

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -47,31 +47,9 @@
       <option name="SPACE_BEFORE_SUPERCLASS_COLON" value="false" />
     </Objective-C>
     <Objective-C-extensions>
-      <option name="GENERATE_INSTANCE_VARIABLES_FOR_PROPERTIES" value="ASK" />
```
Comment (domdomegg) [inline .idea/codeStyles/Project.xml]: Can we revert these changes, so this PR is just the button text changes please?

File: app/src/main/java/fr/free/nrw/commons/review/ReviewImageFragment.java [hunk 2]
```diff
@@ -81,17 +81,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             case SPAM:
                 question = getString(R.string.review_spam);
                 explanation = getString(R.string.review_spam_explanation);
-                yesButtonText = getString(R.string.review_spam_yes_button_text);
-                noButtonText = getString(R.string.review_spam_no_button_text);
+                yesButtonText = getString(R.string.no);
```
Comment (macgills) [inline app/src/main/java/fr/free/nrw/commons/review/ReviewImageFragment.java]: yes is assigned no?

**Comment by madhurgupta10** [comment]
@GearGit Any update on this PR?

**Comment by GearGit** [comment]
> @GearGit Any update on this PR?
I updated as per the requirements

File: .idea/codeStyles/Project.xml [hunk 3]
```diff
@@ -47,6 +47,28 @@
       <option name="SPACE_BEFORE_SUPERCLASS_COLON" value="false" />
     </Objective-C>
     <Objective-C-extensions>
+      <option name="GENERATE_INSTANCE_VARIABLES_FOR_PROPERTIES" value="ASK" />
```
Comment (madhurgupta10) [inline .idea/codeStyles/Project.xml:50]: This file is not relevant to this PR so you can remove it from your version control

File: app/src/main/java/fr/free/nrw/commons/review/ReviewImageFragment.java (hunk 2 above)
Comment (madhurgupta10) [inline app/src/main/java/fr/free/nrw/commons/review/ReviewImageFragment.java:84]: can you assign the **yes button** with text for **Yes** instead of **No** and same for **no button**?

File: .idea/codeStyles/Project.xml (hunk 3 above)
Comment (sivaraam) [reply .idea/codeStyles/Project.xml:50]: We intentionally track that file. So, instead of removing the file itself, just revert the changes made to it.

**Comment by madhurgupta10** [comment]
> @madhurgupta10 Made the changes you asked
Please also revert the changes in .idea/codeStyles/Project.xml

**Comment by sivaraam** [comment]
@GearGit I think it makes sense to change the "Yes, why not" in the Thanks screen to "Yes" too.

Also, remove the strings the strings from `strings.xml` as they are not used anywhere else. Note: It's enough if you just remove the string from the English version. The others would be automatically taken care by Translatewiki.

**Comment by madhurgupta10** [comment]
@GearGit Let us know if you are still working on this.

**Comment by GearGit** [comment]
> @GearGit Let us know if you are still working on this.

I am no longer working on this

**Comment by madhurgupta10** [comment]
> > @GearGit Let us know if you are still working on this.
> 
> I am no longer working on this

Thanks for your quick reply, I am closing this PR! Feel free to comment on the issue thread if you want to work on it again :)
